src/aics-image-viewer/components/App/index.tsx

```diff
@@ -161,9 +160,8 @@ const App: React.FC<AppProps> = (props) => {
 
   // State management /////////////////////////////////////////////////////////
 
-  // TODO is there a better API for values that never change?
   const view3d = useConstructor(() => new View3d());
-  const volumeCache = useConstructor(() => new VolumeCache(250_000_000 * 4));
+  const loadContext = useConstructor(() => new VolumeLoaderContext(250_000_000 * 4, 8, 3));
```

Comment:
turn these numbers into named symbolic constants. Also would be _ideal_ if this app just used the viewer defaults (ie. make these the defaults).